sdk/appconfiguration/azure-appconfiguration/tests/asynctestcase.py

```diff
@@ -14,10 +14,12 @@
 class AsyncAppConfigTestCase(AppConfigTestCase):
     def create_aad_client(self, appconfiguration_endpoint_string):
         cred = self.get_credential(AzureAppConfigurationClient, is_async=True)
-        return AzureAppConfigurationClient(appconfiguration_endpoint_string, cred)
+        return AzureAppConfigurationClient(appconfiguration_endpoint_string, cred, api_version="2022-11-01-preview")
```

Comment:
Why we don't test the GA swagger?

Reply:
It's not available yet.

Reply:
To help me understand better, you mean API version "2023-10-01" is not deployed?

Reply:
Yup. The latest plan I heard in service team is to roll out "2023-10-01" API version on 10/31 in Canary. If everything works well, will deploy to public on 11/6.

Reply:
Using GA API "2023-10-01" now.